processing file asmFiles/branches.asm (1 of 24)
processing file asmFiles/branches.asm~ (2 of 24)
processing file asmFiles/count_days.asm (3 of 24)
processing file asmFiles/branches.asm (1 of 24)
--- memsim.hex	2015-10-14 14:42:40.516864573 -0400
+++ memcpu.hex	2015-10-14 14:21:24.069770400 -0400
@@ -1,7 +1,7 @@
 :040000003401BBBB51
 :040001003402CCCC2D
 :040002003403BBBB4D
-:0400030014220001C2
+:0400030010230001C5
 :04000400AC4100000B
 :04000500AC42000405
 :04000600FFFFFFFFFA
processing file asmFiles/branches.asm~ (2 of 24)
processing file asmFiles/count_days.asm (3 of 24)
--- memsim.hex	2015-10-14 14:42:43.279942736 -0400
+++ memcpu.hex	2015-10-14 14:21:24.069770400 -0400
@@ -1,55 +1,9 @@
-:040000003402016D58
-:04000100340307D0ED
-:040002003404001EA4
-:0400030034050001BF
-:0400040034060014AA
-:0400050034070008B4
-:04000600340807DFD4
-:04000700341CFFFBAB
-:04000800341DFFFCA8
-:04000900341EFFFCA6
-:04000A0027BDFFFC13
-:04000B00AFA800009A
-:04000C0027BDFFFC11
-:04000D00AFA200009E
-:04000E0027BDFFFC0F
-:04000F00AFA300009B
-:0400100027BDFFFC0D
-:04001100AFA7000095
-:0400120027BDFFFC0B
-:04001300AFA4000096
-:0400140027BDFFFC09
-:04001500AFA5000093
-:0400160008000018C6
-:040017000012A8200B
-:04001800341200009E
-:0400190013BE001200
-:04001A008FAD0000A6
-:04001B0027BD0004F9
-:04001C008FAA0000A7
-:04001D0027BD0004F7
-:04001E008FAB0000A4
-:04001F0027BD0004F5
-:04002000016D6023EB
-:04002100000C982017
-:04002200000A582058
-:040023001180000642
-:0400240000AC5824B0
-:040025001160000165
-:04002600024A9020DA
-:0400270001806042B2
-:040028000140504003
-:0400290008000023A8
-:04002A0013BE0002FF
-:04002B0008000017B2
-:04002C00FFFFFFFFD4
-:04002D0002B2A82053
-:04002E0000D5B82021
-:04002F00FFFFFFFFD1
-:043FF90000000001C3
-:043FFA000000001EA5
-:043FFB0000000008BA
-:043FFC00000007D0EA
-:043FFD000000016D52
-:043FFE00000007DFD9
+:040000003401BBBB51
+:040001003402CCCC2D
+:040002003403BBBB4D
+:0400030010230001C5
+:04000400AC4100000B
+:04000500AC42000405
+:04000600FFFFFFFFFA
+:043334000000CCCCFD
 :00000001FF
processing file asmFiles/count_days.asm~ (4 of 24)
processing file asmFiles/branches.asm (1 of 24)
processing file asmFiles/branches.asm~ (2 of 24)
processing file asmFiles/count_days.asm (3 of 24)
processing file asmFiles/count_days.asm~ (4 of 24)
processing file asmFiles/#fib1.asm# (5 of 24)
processing file asmFiles/fib1.asm (6 of 24)
processing file asmFiles/fib1.asm~ (7 of 24)
processing file asmFiles/fib.asm (8 of 24)
processing file asmFiles/jumpandlink.asm (9 of 24)
processing file asmFiles/jumpandlink.asm~ (10 of 24)
processing file asmFiles/mergesort.asm (11 of 24)
processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/test.ldst2.asm (3 of 5)
processing file asmFiles/test.loadstore.asm (4 of 5)
processing file asmFiles/test.rtype.asm (5 of 5)
processing file asmFiles/branches.asm (1 of 24)
processing file asmFiles/branches.asm~ (2 of 24)
processing file asmFiles/count_days.asm (3 of 24)
processing file asmFiles/count_days.asm~ (4 of 24)
processing file asmFiles/#fib1.asm# (5 of 24)
processing file asmFiles/fib1.asm (6 of 24)
processing file asmFiles/fib1.asm~ (7 of 24)
processing file asmFiles/fib.asm (8 of 24)
processing file asmFiles/jumpandlink.asm (9 of 24)
processing file asmFiles/jumpandlink.asm~ (10 of 24)
processing file asmFiles/mergesort.asm (11 of 24)
processing file asmFiles/mult.asm (12 of 24)
processing file asmFiles/mult.asm~ (13 of 24)
processing file asmFiles/mult_procedure.asm (14 of 24)
processing file asmFiles/#orisw.asm# (15 of 24)
processing file asmFiles/orisw.asm (16 of 24)
processing file asmFiles/orisw.asm~ (17 of 24)
processing file asmFiles/search.asm (18 of 24)
processing file asmFiles/#test.branchjump1.asm# (19 of 24)
processing file asmFiles/test.branchjump1.asm (20 of 24)
processing file asmFiles/test.ldst1.asm (21 of 24)
processing file asmFiles/test.ldst2.asm (22 of 24)
processing file asmFiles/test.loadstore.asm (23 of 24)
processing file asmFiles/test.rtype.asm (24 of 24)
processing file asmFiles/branches.asm (1 of 24)
processing file asmFiles/branches.asm~ (2 of 24)
processing file asmFiles/count_days.asm (3 of 24)
processing file asmFiles/count_days.asm~ (4 of 24)
processing file asmFiles/#fib1.asm# (5 of 24)
processing file asmFiles/fib1.asm (6 of 24)
processing file asmFiles/fib1.asm~ (7 of 24)
processing file asmFiles/fib.asm (8 of 24)
processing file asmFiles/jumpandlink.asm (9 of 24)
processing file asmFiles/jumpandlink.asm~ (10 of 24)
processing file asmFiles/mergesort.asm (11 of 24)
processing file asmFiles/mult.asm (12 of 24)
processing file asmFiles/mult.asm~ (13 of 24)
processing file asmFiles/mult_procedure.asm (14 of 24)
processing file asmFiles/#orisw.asm# (15 of 24)
processing file asmFiles/orisw.asm (16 of 24)
processing file asmFiles/orisw.asm~ (17 of 24)
processing file asmFiles/search.asm (18 of 24)
processing file asmFiles/#test.branchjump1.asm# (19 of 24)
processing file asmFiles/test.branchjump1.asm (20 of 24)
processing file asmFiles/test.ldst1.asm (21 of 24)
processing file asmFiles/test.ldst2.asm (22 of 24)
processing file asmFiles/test.loadstore.asm (23 of 24)
processing file asmFiles/test.rtype.asm (24 of 24)
